modules/ivc_champva/spec/models/vha_10_7959c_spec.rb

```diff
@@ -72,10 +71,32 @@
   end
 
   describe '#handle_attachments' do
-    it 'renames the file and returns the new file path' do
-      allow(File).to receive(:rename)
-      result = instance.handle_attachments(file_path)
-      expect(result).to eq(["#{uuid}_vha_10_7959c-tmp.pdf"])
+    context 'Feature champva_unique_temp_file_names=true' do
+      before do
+        Flipper.enable(:champva_unique_temp_file_names)
```

Comment:
Sometimes enabling and disabling Flippers can have unintended side effects in other tests when tests run in parallel. Mocking keeps things isolated. Example: 
```ruby
allow(Flipper.champva_unique_temp_file_names).to receive(:enabled?).and_return(true)
```
Can you update these for the tests? That line would be in the `before do` section.